sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBSuite.scala

```diff
@@ -865,10 +1076,14 @@ class RocksDBSuite extends SparkFunSuite {
     }
   }
 
+  private def sqlConf = SQLConf.get
```

Comment:
I feel it is fine in rocksdb unit test because we modify a small number of configs here. I do it so that I can propagate sql conf of changelog checkpointing conveniently.

Reply:
I'd rather say, clone the config and use cloned one rather than modifying the config in session directly. If it does break any tests I'd consider it as not properly isolated.